spec/features/work_packages/select/select_query_spec.rb

```diff
@@ -83,7 +83,7 @@
     it 'shows the saved filters', js: true do
       filters.open
       filters.expect_filter_by 'Assignee', 'is (OR)', ['me']
-      filters.expect_filter_by 'Progress (%)', '>=', ['10'], 'percentageDone'
+      filters.expect_filter_by 'Percent Complete', '>=', ['10'], 'percentageDone'
```

Comment:
I believe this should be '% Complete'

Reply:
Actually, it does not really matter. In `expect_filter_by`, the name is not used if there is a selector. I was surprised that it was still passing after renaming, so I changed it to something which is not meaningful (does not match selector or translated name) to make it a bit more self-explanatory that this parameter is actually not relevant.